mathics/builtin/functional/apply_fns_to_lists.py

```diff
@@ -415,19 +410,16 @@ class Scan(Builtin):
         "Scan[f_][expr_]": "Scan[f, expr]",
     }
 
-    def eval_invalidlevel(self, f, expr, ls, evaluation, options={}):
-        "Scan[f_, expr_, ls_, OptionsPattern[Map]]"
-
-        evaluation.message("Map", "level", ls)
-
-    def eval_level(self, f, expr, ls, evaluation, options={}):
-        """Scan[f_, expr_, Optional[Pattern[ls, _?LevelQ], {1}],
+    def eval_level(self, f, expr, levelspec, evaluation, options={}):
+        """Scan[f_, expr_, Optional[levelspec_, {1}],
```

Comment:
`levelspec` must be the name of a symbol in Mathics, so cannot follow the snake name convention. `levelspecification` is too long,  and we already have use  `levelspec` in the `python_levelspec` function name.

Reply:
Thanks - that's fine. `ls` is too vague and confusing &mdash; to prove it, I got confused.